sql/moz-fx-data-shared-prod/telemetry_derived/hcm_clients_v1/query.sql

```diff
@@ -1,34 +1,37 @@
+WITH clients AS (
+  SELECT
+    DATE(main.submission_timestamp) AS submission_date,
+    main.client_id,
+    mozfun.stats.mode_last(ARRAY_AGG(main.normalized_channel)) AS channel,
+    mozfun.stats.mode_last(ARRAY_AGG(main.environment.system.os.name)) AS os,
+    mozfun.json.mode_last(ARRAY_AGG(a11y_theme)) AS a11y_theme,
+    main.environment.system.os.name AS os,
```

Comment:
From a maintenance POV, it's looking like we should make sure these are in `clients_daily` and have them query that table directly. We could expose a subset of fields as a new Looker explore.

Reply:
turns out it's already in clients_daily, i just missed it. it was added in https://github.com/mozilla/bigquery-etl/pull/2529, and now appears to already have history at least as far back as 2021-01-01.